src/user/user.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Req, Get, UseGuards } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { Request } from 'express';
+
+@Controller('users')
+export class UserController {
+  @UseGuards(AuthGuard())
+  @Get('me')
+  getMyProfile(@Req() req: Request): any {
```

Comment:
- Add response type
- Maybe, we should fetch DB to bring user data from the table. Because in the future, project might have more columns that the JWT payload.